samples/app-sso/csharp/app_sso/Resources/justintimeinstallation.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "AdaptiveCard",
```

Comment:
We don't need justinTime installation card here please remove this, Let me know if it is being used